src/components/with-grouped-children-hoc.js

```diff
@@ -0,0 +1,16 @@
+import { h } from 'preact';
+
+export function withGroupedChildren(Component) {
```

Comment:
I didn't know you had access to private Wix repos...

Reply:
😂